keras_nlp/models/gpt_neo_x/gpt_neo_x_tokenizer_test.py

```diff
@@ -99,7 +100,6 @@ def test_serialization(self):
         )
 
     @pytest.mark.large
-    @pytest.mark.tf_only
     def test_saved_model(self):
```

Comment:
leave the tf_only here (due to `keras.Input(dtype="string", shape=())` failing on jax and torch).